### ISSUE TITLE
Cannot scroll on listings pages

### ISSUE THREAD

**Issue by sksiyer** [thepracticaldev/dev.to#7786]
**Describe the bug**
I can't scroll when in listings page

**To Reproduce**
Go to listings page and select a category, at the moment the collabs listing has a number of items that can't be accessed
https://dev.to/listings/collabs

**Additional context**
I couldn't help but have a look under the hood. It looks like the .modal-open CSS class is on the page body. This class has attribute overflow: hidden, as expected for modals but this a page not a modal :)

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by nickytonline** [comment]
@sksiyer, thanks for opening this issue.

I've tested this in production and paging does indeed work. If you select All category or just navigate to https://dev.to/listings and scroll to the bottom, you will see the button to load more listings.

![image](https://user-images.githubusercontent.com/833231/81852116-86bd3a00-9528-11ea-89d6-7b43fe00142e.png)

I checked and had our SRE double check how many published listings we currently have in production and it's 83. The Contributors/Collaborators Wanted category currently only has 8 listings.

![image](https://user-images.githubusercontent.com/833231/81852453-0b0fbd00-9529-11ea-9692-240b88544975.png)

I know we had more listings recently, but a bunch must have expired.
